components/dashboard/src/workspaces/PersonalizedContent.tsx

```diff
@@ -0,0 +1,260 @@
+            jobRole: ["enabling", "team-lead"],
+            explorationReasons: ["replace-remote-dev"],
+            signupGoals: ["efficiency-collab", "security"],
+        },
+    },
+    {
+        url: "https://www.gitpod.io/blog/whats-a-cloud-development-environment",
+        title: "What's a CDE",
+        label: "what-is-cde",
+        priority: 9,
+        recommended: {
+            jobRole: ["enabling", "team-lead"],
+        },
+    },
+];
+
+const defaultContent: ContentItem[] = [
+    {
+        url: "https://www.gitpod.io/blog/whats-a-cloud-development-environment",
+        title: "What's a CDE",
+        label: "what-is-cde",
+    },
+    {
+        url: "https://www.gitpod.io/solutions/onboarding",
+        title: "Onboarding developers in one click",
+        label: "onboarding-solutions",
+    },
+    {
+        url: "https://www.gitpod.io/blog/using-a-cde-roi-calculator",
+        title: "Building a business case for Gitpod",
+        label: "cde-roi-calculator",
+    },
+];
+
+const PersonalizedContent: React.FC = () => {
+    const user = useCurrentUser();
+    const [selectedContent, setSelectedContent] = useState<ContentItem[]>([]);
+
+    useEffect(() => {
+        if (!storageAvailable("localStorage")) {
+            // Handle the case where localStorage is not available
+            setSelectedContent(getFirstWeekContent(user));
+            return;
+        }
+
+        let content: ContentItem[] = [];
+        let lastShownContent: string[] = [];
+
+        try {
+            const storedContentData = localStorage.getItem("personalized-content-data");
+            const currentTime = new Date().getTime();
+
+            if (storedContentData) {
+                const { lastTime, lastContent } = JSON.parse(storedContentData);
+                const WEEK_IN_MS = 7 * 24 * 60 * 60 * 1000;
+                const weeksPassed = Math.floor((currentTime - lastTime) / WEEK_IN_MS);
+                lastShownContent = lastContent || [];
+
+                if (weeksPassed >= 1) {
+                    content = getRandomContent(contentList, 3, lastShownContent);
+                } else {
+                    content = getFirstWeekContent(user);
+                }
+            } else {
+                content = getFirstWeekContent(user);
+            }
+
+            localStorage.setItem(
+                "personalized-content-data",
+                JSON.stringify({
+                    lastContent: content.map((item) => item.label),
+                    lastTime: currentTime,
+                }),
+            );
+
+            setSelectedContent(content);
+        } catch (error) {
+            console.error("Error handling personalized content: ", error);
+            setSelectedContent(getRandomContent(contentList, 3, []));
+        }
+    }, [user]);
+
+    return (
+        <div className="flex flex-col gap-2">
+            <h3 className="text-lg font-semibold text-pk-content-primary">Personalised for you</h3>
```

Comment:
This looks like a nice candidate for swapping out with Podkit's [`<Heading3 />`](https://github.com/gitpod-io/gitpod/blob/4bccf5282c00602187da1e16f7cdab06876339cd/components/dashboard/src/components/typography/headings.tsx#L44-L53) to remove style duplication.

Reply:
Resolved in [6041f16](https://github.com/gitpod-io/gitpod/pull/20034/commits/6041f1686e05cf58600b30dad65be1ffdb82a571)